src/netif.c

```diff
@@ -709,6 +711,18 @@ static void cpu_id_handler(vector_t tokens)
     FREE_PTR(str);
 }
 
+#ifdef CONFIG_ICMP_FWD_CORE
+static void cpu_icmp_fwd_handler(vector_t tokens)
+{
+    struct worker_conf_stream *current_worker = list_entry(worker_list.next,
+            struct worker_conf_stream, worker_list_node);
+
+    RTE_LOG(INFO, NETIF, "%s(%d) used to forward icmp\n",
```

Comment:
"%s(%d) **is** used to **redirect** icmp packets

Reply:
Has been modified